examples/xgboost_cv.py

```diff
@@ -0,0 +1,104 @@
+"""
+Optuna example that performs Cross Validation for cancer dataset using XGBoost.
+
+In this example, we perform cross-validation to accuracy of cancer detection
+using XGBoost. We optimize both the choice of booster model and their hyper
+parameters.
+
+We have following two ways to execute this example:
+
+(1) Execute this code directly.
+    $ python xgboost_cv.py
+
+
+(2) Execute through CLI.
+    $ STUDY_NAME=`optuna create-study --direction maximize --storage sqlite:///example.db`
+    $ optuna study optimize xgboost_cv.py objective --n-trials=100 --study $STUDY_NAME \
+      --storage sqlite:///example.db
+
+"""
+
+
+import os
+import shutil
+import sklearn.datasets
+import sklearn.metrics
+import xgboost as xgb
+
+import optuna
+
+
+# Set constants
+SEED = 108
+N_FOLDS = 3
+CV_RESULT_DIR = "./xgboost_cv_results"
+
+if not os.path.exists(file_path):
```

Comment:
`file_path` is not defined in this scope. I think it should be `CV_RESULT_DIR`.
```suggestion
if not os.path.exists(CV_RESULT_DIR):
```